packages/adapters/src/docker-instance-adapter.ts

```diff
@@ -26,20 +28,22 @@ class DockerInstanceAdapter implements
 ILifeCycleAdapterMain,
 ILifeCycleAdapterRun,
 IComponent {
-    private dockerHelper: IDockerHelper;
+    private dockerHelper: DockerodeDockerHelper;
```

Comment:
```IDockerHelper``` was designed to easy replace ```Dockerode``` with other lib to communicate with Docker.
What is the reason for change ?

Reply:
it was just for faster iterations. I will refactor it so it's encapsulated in the Interface